governance.md

```diff
@@ -0,0 +1,145 @@
+
+Users who continue to engage with the project and its community will often become more and more involved. Such Users may find themselves becoming [Contributors](#Contributors), as described in the next section.
+
+### Contributors
+Contributors are community members who contribute in concrete ways to the project, most often in the form of data updates, code and/or documentation. Anyone can become a Contributor, and contributions can take many forms. There is no expectation of commitment to the project, no specific skill requirements, and no selection process.
+
+Contributors:
+- Have read-only access to source code and therefore can submit changes via pull requests.
+- Have their contribution reviewed and merged by a [Peer](#Peers) or [Owner](#Owners). Owners and Peers work with Contributors to review their code and prepare it for merging.
+- May also review pull requests. This can be helpful, but their approval or disapproval is not decisive for merging or not merging PRs.
+
+As Contributors gain experience and familiarity with the project, their profile within, and commitment to, the community will increase. At some stage, they may find themselves being nominated for becoming a Peer by an existing Peer or Owner.
+
+### Peers
+Peers are community members who have shown that they are committed to the continued development of the project through ongoing engagement with the community. Peers are given push/write access to the project’s GitHub repos.
+
+Peers:
+
+- Are expected to work on public branches of their forks and submit pull requests to the master branch.
+- Must submit pull requests for all their changes.
+- May label and close issues.
+- May only merge other people's pull requests that relate to compat data updates.
+- Have their non-data update work reviewed and merged by [Owners](#Owners). Non-data pull requests are PRs that change the schema, update project meta-docs, the linter, or other infrastructure changes.
+- Should ask for additional review from other Peers or Owners on other people's PRs that are disruptive or controversial.
+
+To become a Peer one must:
+
+- Have shown a willingness and ability to participate in the project as a team player. Typically, a potential Peer will need to show that they have an understanding of and alignment with the project, its objectives, and its strategy.
+- Have contributed a significant amount of work to the project (e.g. in the form of PRs or PR reviews), thereby demonstrating their trustworthiness and commitment to the project.
+- Read and agree to abide by the [Mozilla Commit Access Requirements](https://www.mozilla.org/en-US/about/governance/policies/commit/requirements/).
+- File an issue in the [mdn/mdn](https://github.com/mdn/mdn) repository and record “I have read, and agree to abide by, the Mozilla Commit Access Requirements.”
```

Comment:
An alternate would be to have an issue in this repo, or even skip the issue and just make it a PR. For the ADRs, we made "approval" a commit against the PR (see the history for [ADR 002](https://github.com/mdn/mdn/commits/master/ADRs/002-use-flow-for-typechecking.md). In a similar way, a user could add their name to the "List of peers" below as a PR, with the required statement as the PR description or commit comment. This would make ``git blame`` useful for cross-referencing.

But, mdn/mdn is fine as well. It is easy to link to an issue in another repo.

Reply:
I believe mdn/mdn is a good place for this. I also think that we would only require to record this statement once in case people apply for being a peer on multiple repos. Maybe mdn/mdn is a good place to keep record of all peers and owners and applications throughout our org. I hope to use it more as MDN makes its way to GitHub.